app/views/editor/level/LevelEditView.coffee

```diff
@@ -210,3 +219,9 @@ module.exports = class LevelEditView extends RootView
     return if application.userIsIdle
     @levelBuildTime ?= @level.get('buildTime') ? 0
     ++@levelBuildTime
+
+  getTaskCompletionRatio: ->
+    if !@level.get('tasks')?
+      return {}
```

Comment:
When a level doesn't have tasks, it looks like this:
<img width="589" alt="screen shot 2016-01-21 at 9 56 34 am" src="https://cloud.githubusercontent.com/assets/204723/12489473/6017726c-c025-11e5-97c3-8cf820897d40.png">